product-development/README.md

```diff
@@ -33,6 +33,35 @@ Anyone advocating for an epic's priority (or priorities in general) should @-men
 
 <img src="images/epics-priority.png" height="100" />
 
+### Estimating Stories
+
+Right now we have Clubhouse configured to use the following, roughly Fibonacci sequence story point estimate scale:
+
+* 1
+* 2
+* 3
+* 5
+* 8
+* 13
+* 21
+
+We use story points for one purpose: to communicate with each other about the complexity of work. As an engineer this is useful when it's 4 PM and I'm looking at an epic I'm working on it's helpful to know if a story is _expected_ to be simple or complicated. As a PM/EM/CEO this is useful when I want to ask questions like "roughly how much work is left on X" and "when should we start work on marketing for Y".
+
+Put another way: we use story points to communicate expectations, to our peers, and to those who operate at a higher level in the company.
```

Comment:
Whose responsibility is it to assign points to a story -- the person who files it? The person who claims it? Some fuzzy combination? (I suspect the latter, but maybe good to lay out an expectation, e.g. "you should always add an estimate to a ticket on creation IF you have the context to do so" or similar)